docs/cli/kops_get.md

```diff
@@ -21,9 +21,12 @@ Display one or many resources.
   # Get a cluster
   kops get cluster k8s-cluster.example.com
   
-  # Get a cluster YAML cluster spec
+  # Get a cluster YAML configuration
```

Comment:
I would probably stick here with cluster specification, IMO.

`kubectl get` just says e.g. "# List a single pod in JSON output format.", so avoids the whole question.